### DISCUSSION TITLE
hashed classnames

### DISCUSSION THREAD

**Discussion by dingram94** [vercel/next.js#51211] [Help] [answered]
### Summary

I am currently setting up a dark mode. We are using plain scss modules and a own custom ui library. I managed to set up the colors on the ui lib so that dark mode looks like intented. There are some things on the next 13 project I want to have differnet css than on light mode but I cant override the classes and check for "dark" class on the html because our classes get hashed. Is there a way to not hash the classes? What is the downside of that approach other than the danger to have duplicate classnames?

We are using simple next config. Can anyone help me to remove hashed classnames?

**Comment by icyJoseph** [comment, marked as answer]
If you want a class to be global, you use `:global(dark-mode-className)` in your CSS/SASS Modules definitions.

The downside is that CSS' global nature means that any other `dark-mode-className` declaration will also apply. That's why CSS/SASS Modules hash the class name, to give it a "scope", which just means it won't crash with other modules, or global stuff.

- https://github.com/css-modules/css-modules#exceptions
